test/functional/test_runner.py

```diff
@@ -255,6 +238,7 @@
     'wallet_keypool.py --descriptors',
     'wallet_descriptor.py --descriptors',
     'p2p_nobloomfilter_messages.py',
+    TEST_FRAMEWORK_UNIT_TESTS,
```

Comment:
is there any reason we picked this location in `BASE_SCRIPTS`, I would think we would want to run `TEST_FRAMEWORK_UNIT_TESTS` earlier on even if they are running in parallel 

since if the unit tests fail other tests will probably fail as well.

I think making it the first script in `BASE_SCRIPTS` would make sense to me and then making an exception in regards to run times since other tests deepened on this.

Reply:
Thank you for the review!

Great question.  The location chosen was within the `# vv Tests less than 30s vv` portion of `BASE_SCRIPTS`.  I'm seeing run-times for `feature_framework_unit_tests.py` (on two machines I'm using) of 9s and 6s, and `TEST_FRAMEWORK_UNIT_TESTS` is around mid-way in the "30s" portion of the list, which seemed reasonable.
https://github.com/bitcoin/bitcoin/blob/f19f0a2e5af6c2a64900f1f229e21b6f1668bd3d/test/functional/test_runner.py#L392-L393

The existing comment to insert at random seems to provide general high-level guidance and doesn't dig into individual system performance (for example, 30s worth of work on an Ivy Bridge CPU and a Zen 4 CPU would be drastically different due to significant IPC differences).  Maybe I'm overlooking other guidance that defines relative performance (e.g. 30s on a particular cloud service instance type, CPU type, etc.).  There's probably an opportunity for rethinking and updating this guidance, but that would be adjacent to this PR rather than included in it.